configure

```diff
@@ -3749,7 +3748,7 @@ check_install_dir
 
 # check for Mathematica if meta code generation is enabled or we
 # compile with LibraryLink
-if [ "$enable_meta" = yes -o \( "$enable_compile" = yes -a "$enable_librarylink" != no \) ]; then
+if [ "${enable_meta}" = yes ] || [ \( "$enable_compile" = yes -a "$enable_librarylink" != no \) ]; then
```

Comment:
I'm wondering why there is a warning on this line. Can you please just paste the warning here?

Reply:
`SC2166: Prefer [ p ] || [ q ] as [ p -o q ] is not well defined.`
and
`SC2166: Prefer [ p ] && [ q ] as [ p -a q ] is not well defined.`

Reply:
I will fix this warning later, after all errors are fixed.

Reply:
Thanks! I've just found that `-o` and `-a` are deprecated:
https://pubs.opengroup.org/onlinepubs/9699919799/utilities/test.html